webknossos/webknossos/geometry/mag.py

```diff
@@ -1,87 +1,112 @@
 import re
 from functools import total_ordering
 from math import log2
-from typing import Any, List
+from typing import Any, List, Optional, Tuple, cast
 
+import attr
 import numpy as np
 
+from .vec3_int import Vec3Int, Vec3IntLike
+
+
+def import_mag(mag_like: Any) -> Vec3Int:
+    as_vec3_int: Optional[Vec3Int] = None
+
+    if isinstance(mag_like, int):
+        as_vec3_int = Vec3Int(mag_like, mag_like, mag_like)
+    elif isinstance(mag_like, Vec3Int):
+        as_vec3_int = mag_like
+    elif isinstance(mag_like, list) or isinstance(mag_like, tuple):
+        as_vec3_int = Vec3Int(cast(Vec3IntLike, mag_like))
```

Comment:
Using a stricter typing for `mag_like`, this should not be necessary anymore (I hope):
```suggestion
        as_vec3_int = Vec3Int(mag_like)
```